# HG changeset patch
# User Rik <rik@octave.org>
# Date 1478297254 25200
#      Fri Nov 04 15:07:34 2016 -0700
# Branch stable
# Node ID 37e53505e2eb4202f9431db3f0bb07c279e1a2b4
# Parent  61179451633266a9165810041e67301a47e29b5f
doc: Use Texinfo dots macro to create ellipses.

* file-io.cc (Fferror): Use @dots Texinfo macro instead of keyword "dots".

diff --git a/libinterp/corefcn/file-io.cc b/libinterp/corefcn/file-io.cc
--- a/libinterp/corefcn/file-io.cc
+++ b/libinterp/corefcn/file-io.cc
@@ -2598,17 +2598,17 @@ end-of-file condition.
 
   return ovl (os.eof () ? 1.0 : 0.0);
 }
 
 DEFUNX ("ferror", Fferror, args, ,
         doc: /* -*- texinfo -*-
 @deftypefn  {} {@var{msg} =} ferror (@var{fid})
 @deftypefnx {} {[@var{msg}, @var{err}] =} ferror (@var{fid})
-@deftypefnx {} {[@var{dots}] =} ferror (@var{fid}, "clear")
+@deftypefnx {} {[@dots{}] =} ferror (@var{fid}, "clear")
 Query the error status of the stream specified by file descriptor @var{fid}
 
 If an error condition exists then return a string @var{msg} describing the
 error.  Otherwise, return an empty string @qcode{""}.
 
 The second input @qcode{"clear"} is optional.  If supplied, the error
 state on the stream will be cleared.
 
